WcaOnRails/app/assets/stylesheets/edit_schedule.scss

```diff
@@ -0,0 +1,223 @@
+      }
+    }
+  }
+
+  #schedules-edit-panel {
+    #activity-picker-panel {
+      &.affix {
+        top: 10px;
+      }
+      &.affix-bottom {
+        position: absolute;
+        bottom: auto;
+      }
+      .selected-activity {
+        border: 2px solid #791f82;
+        margin-top: 1px;
+        margin-bottom: 1px;
+      }
+      .panel-heading {
+        @extend .text-center;
+      }
+      .panel-body {
+        padding: 5px;
+        overflow-y: auto;
+      }
+      .event-picker-line {
+        .row {
+          min-height: 32px;
+          .activity-icon {
+            padding: 0;
+            .cubing-icon {
+              font-size: 22px;
+              &::before {
+                margin-top: 2px;
+              }
+            }
+          }
+          .activity-in-picker {
+            padding: 0;
+          }
+        }
+      }
+    }
+    .room-selector {
+      label {
+        padding-top: 7px;
+      }
+    }
+    #schedule-editor {
+      #schedule-calendar {
+        &:target {
+          // Override the flashy yellow we use for targets elsewhere
+          background-color: transparent;
+        }
+
+        .fc-helper-container > .fc-event,
+        .fc-event.selected-fc-event {
+          border: 2px solid #791f82;
+        }
+
+        .fc-time-grid-event.fc-short .fc-time span {
+        }
+
+        .fc-time-grid-event.fc-short .fc-time::before {
+        }
+
+        .fc-event {
+          &.fc-short > .fc-content {
+            // By default for vertically short events FC don't display the full time
+            // But we have some width to do so
+            .fc-time {
+              &::before {
+                content: "";
+              }
+              &::after {
+                content: ":\000A0";
+              }
+              span {
+                display: inline-block;
+              }
+            }
+          }
+          .fc-bg {
+            display: none;
+          }
+        }
+      }
+      #drop-event-area {
+        z-index: 0;
+        width: 100%;
+        min-height: 40px;
+        line-height: 36px;
+        padding-left: 5px;
+        padding-right: 5px;
+        margin-bottom: 15px;
+        border-radius: 0.25em;
+        border: 2px dashed #a94442;
+        &.event-on-top {
+          border-color: #f2dede;
+          color: #f2dede;
+          background-color: #a94442;
+        }
+        .fa {
+          font-size: 20px;
+          margin-left: 15px;
+          margin-right: 15px;
+          line-height: 36px;
+        }
+      }
+    }
+  }
+}
+#tooltip-enable-keyboard {
+  .tooltip-inner {
+    max-width: none;
+  }
+}
+
+#calendar-settings-popover {
+  .setting-label {
+    padding-top: 7px;
+    padding-right: 5px;
+    padding-left: 0;
+    font-size: 12px;
+    text-align: right;
+  }
+  select {
+    @extend .input-sm;
+    margin-bottom: 5px;
+  }
+}
+
+// This needs to be here, to overcome an issue with draggable activity + overflow container
+.activity {
```

Comment:
There's a *lot* of css in this file. That's ok if it's necessary, but is it possible to keep it all inside of the `#edit-schedule-area`, just so we don't have to worry about it affecting other pages?